client/redux/artwork.js

```diff
@@ -20,7 +20,7 @@ export const getArtwork = (id) => {
   };
 
 
-export default function artworkReducer( state = [], action) {
+export default function artworkReducer( state = {} , action) {
```

Comment:
Thanks for changing! I think I left a comment on this a while back but never checked on it